Code/src/main/java/nl/utwente/group10/ui/BackendUtils.java

```diff
@@ -31,4 +33,18 @@ public static Type dive(ConstT function, int steps) {
         }
         return function;
     }
+    
+    /**
+     * @return Whether or not t1 and t2 unify with each other.
```

Comment:
You are describing a different problem than the one the method is supposed to solve.
I dont need to infer the Type anymore, I already have 2 types. I just need to know whether they match or not.

Possibly all of this will change when having talked with @jjkester , but as it stands this is how it is used.